### PR TITLE
fix(theme): fix the abandoned normal level in the theme file

### PR DESCRIPTION
**Pull request by Liangism**
## Brief Information

This pull request is in the type of:

- [x] bug fixing
- [ ] new feature
- [ ] others

### Fixed issues

- #17473

## Document Info

One of the following should be checked.

- [x] This PR doesn't relate to document changes
- [ ] The document should be updated later
- [ ] The document changes have been made in apache/echarts-doc#xxx

### ZRender Changes

- [ ] This PR depends on ZRender changes (ecomfe/zrender#xxx).

### Related test cases or examples to use the new APIs

N.A.

### Merging options

- [ ] Please squash the commits into a single one when merging.

### CODE REVIEW

File: theme/macarons2.js [hunk 1]
```diff
@@ -89,69 +89,66 @@
...
+                // color: '各异',
+                barBorderColor: '#fff', // 柱条边线
+                barBorderRadius: 0, // 柱条边线圆角，单位px，默认为0
+                barBorderWidth: 1, // 柱条边线线宽，单位px，默认为1
+                label: {
+                    show: false
+                    // position: 默认自适应，水平布局为'top'，垂直布局为'right'，可选为
+                    //           'inside'|'left'|'right'|'top'|'bottom'
+                    // textStyle: null      // 默认使用全局文本样式，详见TEXTSTYLE
+                }
```
Comment (plainheart) [inline theme/macarons2.js:101]: The `label` block should be moved to the outside.

File: theme/macarons2.js [hunk 2]
```diff
@@ -186,15 +183,15 @@
...
-                },
-                labelLine: {
-                    show: true,
-                    length: 20,
-                    lineStyle: {
-                        // color: 各异,
-                        width: 1,
-                        type: 'solid'
-                    }
+                }
```
Comment (plainheart) [inline theme/macarons2.js:186]: The `label` block should be moved to the outside as well.

**Review by plainheart — COMMENTED** [review]
Generally, it looks good to me. Left several minor issues.

**Review by Ovilia — DISMISSED** [review]
LGTM

File: theme/macarons2.js (hunk 2 above)
Comment (Liangism) [reply theme/macarons2.js]: Okay, I changed it again.😁

**Review by plainheart — APPROVED** [review]
Looks good! Thanks for your contribution!

**Pull request merged**